Source/EasyNetQ/Producer/Rpc.cs

```diff
@@ -171,7 +171,11 @@ protected void RegisterErrorHandling<TResponse>(Guid correlationId, Timer timer,
 
                 var exchange = DeclareRpcExchange(conventions.RpcResponseExchangeNamingConvention(responseType));
```

Comment:
I would move this declaration inside the below check, checking on the string returned by `conventions.RpcResponseExchangeNamingConvention(responseType)`, and declare the exchange only if it's not the default, so we can avoid the change on advanced api that I don't like